source/material_model/drucker_prager.cc

```diff
@@ -224,6 +224,7 @@ namespace aspect
                              "Units: $W/m/K$.");
           prm.enter_subsection ("Viscosity");
           {
+
```

Comment:
@naliboff, I don't know if this causes the problem, but you want to remove the Angle of internal friction/Cohesion below (and same in parse parameters). These things are now read in the Rheology::DruckerPrager<dim>::declare_parameters(prm) function, right?

Reply:
@jdannberg - Thanks for taking a look! I should have added a comment on Friday after figuring out the solution. 

I actually intentionally left `Angle of internal friction` and `Cohesion` in `drucker_prager` as  using the equivalent parameters in `Rheology::DruckerPrager` would have required adding a decent amount of new code and adjusting tests. This is due to the  parameters in `Rheology::DruckerPrager` being lists, rather than single values.

@gassmoeller and I talked briefly on Friday about this and the issue is due to not having simulator access in the` drucker_prager_derivatives` test, which we need to get the number of compositional fields when parsing parameters. I checked with @MFraters and the correct path forward is to adjust the test (or make new tests) following the `visco_plastic_derivatives_` tests, which have simulator access. This does indeed work and I'll submit the new tests in a separate PR.